lib/dartdoc.dart

```diff
@@ -252,12 +252,13 @@ class DartDoc {
     while (result.hasMoreWork) {
       result = context.performAnalysisTask();
     }
-
+    
     // Use the includeExternals.
     for (Source source in context.librarySources) {
       LibraryElement library = context.computeLibraryElement(source);
```

Comment:
This and libraryName are no longer used in the case where the library doesn't match.  No clue if it's expensive or not.  (Sorry to keep commenting on your completed CL, was just playing around in this code and noticed.)  Thanks again for the fix! :)